### ISSUE TITLE
Libdivide is using exit instead of abort on errors.

### ISSUE THREAD

**Issue by alexey-milovidov** [ridiculousfish/libdivide#64]
```
#define LIBDIVIDE_ERROR(msg) \
    do { \
        fprintf(stderr, "libdivide.h:%d: %s(): Error: %s\n", \
            __LINE__, LIBDIVIDE_FUNCTION, msg); \
        exit(-1); \
    } while (0)
```

The usage of `exit` is wrong for two reasons:
1. It does not quickly terminate the program because it calls global destructors (in contrast to `_exit`).
2. Core won't be dumped.

The issue has been found here: https://github.com/ClickHouse/ClickHouse/issues/12119

**Comment by ridiculousfish** [comment]
Yeah well found, it should be fixed! If you make a PR I will merge it!

**Comment by alexey-milovidov** [comment]
Ok https://github.com/ridiculousfish/libdivide/pull/65

**Comment by ridiculousfish** [comment]
Thank you! Closed in #65
